### PR TITLE
added const secday, which is useful for binary stuff

### PR DESCRIPTION
**Pull request by matthiasfabry**
as the title says.

### CODE REVIEW

File: const/public/const_def.f90 [hunk 1]
```diff
@@ -94,7 +94,9 @@ module const_def
       real(dp), parameter :: au = 1.49597870700D13 ! (cm) - exact value defined by IAU 2009, 2012
       real(dp), parameter :: pc = (3.600D3 * rad2a) * au ! (cm) parsec, by definition
       real(dp), parameter :: dayyer = 365.25d0 ! days per (Julian) year
-      real(dp), parameter :: secyer = 24*60*60*dayyer ! seconds per year
+      real(dp), parameter :: secyer = secday*dayyer ! seconds per year
```
Comment (warrickball) [inline const/public/const_def.f90:97]: Duplicate of line 99, no?

**Comment by warrickball** [comment]
On balance, I think this is a sensible change. I expect most people recognise factors like `24*60*60` and `86400` as the number of seconds in a day but using `secday` is just that little bit clearer and slightly less error prone (e.g. `.../24*60*60` instead of `.../(24*60*60)`). I'd suggest we wait until early next week to give other devs a chance to respond.

If this is worth changing, you might want to make some substitutions now. Grepping for `"\*60\*"` or `"\*60d0\*"` in the source (including test cases) turns up some easy replacements.

**Comment by orlox** [comment]
A bit of a tangential thing, but how would people feel about changing `secyer` to `secyear`? Guess it would break a bit of external code, but in a pretty trivial way that compilers should be able to suggest how to fix.

**Comment by fxt44** [comment]
or even the more obvious second_per_year. neither here nor there, but the secyer naming is from ~2005.

**Comment by evbauer** [comment]
Personally, I feel like a "fix" like `secyear` will just get us into [bikeshedding](https://en.wiktionary.org/wiki/bikeshedding)  territory, so I'd prefer just to leave it.

(But I fully approve of `secday`.)

**Review by rjfarmer — APPROVED** [review]

**Pull request merged**